addons/survey/models/survey_question.py

```diff
@@ -447,6 +448,7 @@ class SurveyQuestionAnswer(models.Model):
     matrix_question_id = fields.Many2one('survey.question', string='Question (as matrix row)', ondelete='cascade')
     sequence = fields.Integer('Label Sequence order', default=10)
     value = fields.Char('Suggested value', translate=True, required=True)
+    image = fields.Image('Image', max_width=256, max_height=256)
```

Comment:
Can you rename to `question_image` ?

For future grepping, "image" will give too many results.

Reply:
done, I renamed it to answer_image since it's linked to an answer and not a question. It allows make more sense with the previous change where answer_image was replaced by allow_answer_image.